geode-core/src/distributedTest/java/org/apache/geode/management/internal/security/MultiUserAuthenticationDUnitTest.java

```diff
@@ -100,4 +112,65 @@ public void multiAuthenticatedView() throws Exception {
       cache.close();
     }
   }
+
+  @Test
+  public void multiUserCQ() throws Exception {
```

Comment:
that's covered by other tests in the same class. I added this specific test to verify a bug I fixed in this PR. (before this test would fail with "subject not found" exception)